Package.swift

```diff
@@ -0,0 +1,27 @@
+// swift-tools-version:5.2
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "OneSignal",
+    products: [
+        .library(
+            name: "OneSignal",
+            type: .static,
```

Comment:
I noticed this in the swift docs:
> /// Create a library product that can be used by clients that depend on this package.
///
/// A library's product can either be statically or dynamically linked. It
/// is recommended to not declare the type of library explicitly to let the
/// Swift Package Manager choose between static or dynamic linking depending
/// on the consumer of the package.

Should we remove the `OneSignalDynamic` section and this ` type: .static,` line so Xcode can automatically pick?

Reply:
fixed!

Reply:
Fixed in the new PR
https://github.com/OneSignal/OneSignal-iOS-SDK/pull/712/files#diff-37ca2dd15ca0f6b1b49e78db084ef5b9R6-R21